lib/smart_answer/calculators/self_assessment_penalties.rb

```diff
@@ -42,8 +47,10 @@ def one_year_after_start_date_for_penalties
         Date.new(2015, 2, 01)
       elsif tax_year == '2013-14'
         Date.new(2016, 2, 01)
-      else
+      elsif tax_year == '2014-15'
```

Comment:
I would also add a test to this method: https://github.com/alphagov/smart-answers/blob/07ff1d0751fc5841ec9baffc7eea82fc7d63d0d1/lib/smart_answer/calculators/self_assessment_penalties.rb#L38-L48